dashboard/client/src/components/EventTableMockData.ts

```diff
@@ -0,0 +1,99 @@
+const MESSAGES = [
+  "Serve App {app name} started. Serve App {app name} started. Serve App {app name} started.  {app name} started. Serve App {app name} started. {app name} started. Serve App {app name} started. ",
+  "Serve App {app name} deleted. {app name} started. Serve App {app name} started. {app name} started. Serve App {app name} started.",
+  "Deployment {deployment name} created {app name} started. Serve App {app name} started. {app name} started. Serve App {app name} started.",
+  "Deployment {deployment name} is running {app name} started. Serve App {app name} started. {app name} started. Serve App {app name} started.",
+  "Deployment {deployment name} deleted. {app name} started. Serve App {app name} started. {app name} started. Serve App {app name} started.",
+  "Serve Deployment “job” started. {app name} started. Serve App {app name} started. {app name} started. Serve App {app name} started.",
+  "Replica {replica name} started {app name} started. Serve App {app name} started. {app name} started. Serve App {app name} started.",
+  "Replica {replica name} died",
+  "Ray Serve server is running.",
+  "Ray Serve server is terminated.",
+  "Ray Serve HTTP Proxy started on node {node id}, {node ip}",
+  "Ray Serve HTTP Proxy removed from node {node id}, {node ip}",
+];
+
+let longMessage = "";
+const targetLength = 500;
+
+while (longMessage.length < targetLength) {
+  longMessage += MESSAGES.join("");
+  if (longMessage.length >= targetLength) {
+    longMessage = longMessage.substring(0, targetLength);
+  }
+}
+
+console.log(longMessage);
+
+const CUSTOM_MESSAGES = [
+  "Terminating instance i-xxx due to compute config live update.",
+];
+
+function generateRandomEvent() {
+  const severity = SEVERITIES[Math.floor(Math.random() * SEVERITIES.length)];
+  const source = SOURCES[Math.floor(Math.random() * SOURCES.length)];
+  const message =
+    source === "AUTOSCALER"
+      ? CUSTOM_MESSAGES[0]
+      : MESSAGES[Math.floor(Math.random() * MESSAGES.length)];
+  const customFields = {} as {
+    [key: string]: any;
+  };
+
+  if (message.includes("{app name}")) {
+    const appName = "my-app-" + Math.floor(Math.random() * 100);
+    customFields.app_name = appName;
+  }
+
+  if (message.includes("{deployment name}")) {
+    const deploymentName = "deployment-" + Math.floor(Math.random() * 100);
+    customFields.deployment_name = deploymentName;
+  }
+
+  if (message.includes("{replica name}")) {
+    const replicaName = "replica-" + Math.floor(Math.random() * 100);
+    customFields.replica_name = replicaName;
+  }
+
+  if (message.includes("{node id}") && message.includes("{node ip}")) {
+    const nodeId = "node-" + Math.floor(Math.random() * 100);
+    const nodeIp = "192.168.1." + Math.floor(Math.random() * 256);
+    customFields.node_id = nodeId;
+    customFields.node_ip = nodeIp;
+  }
+
+  return {
+    eventId: Math.random().toString(36).substring(7),
+    sourceType: source,
+    hostName: "po-dev.inc.alipay.net",
```

Comment:
let's use ray-project instead of alipay?

Reply:
Good catch, I did not pay attention to this file since this is the file to create MOCK_DATA and could be deleted after this project is launched.